gradio/_simple_templates/simpledropdown.py

```diff
@@ -23,8 +24,8 @@ def __init__(
         choices: list[str | int | float | tuple[str, str | int | float]] | None = None,
         *,
         value: str | int | float | Callable | None = None,
-        label: str | None = None,
-        info: str | None = None,
+        label: str | TranslationMetadata | None = None,
```

Comment:
Could we give this class a name like `I18nData` instead of `TranslationMetadata`? Only because this is visible to users in the docs and I think we should choose a name that they assiociate with `gr.I18n()`